apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -144,31 +152,6 @@ public static void main(String[] args) throws InterruptedException {
      */
   }
 
-  /**
-   * @param options the {@link DatabaseOptions} to use
-   * @param maxPoolSize the database max pool size
-   * @param metrics the {@link MetricRegistry} to use
-   * @return a {@link HikariDataSource} for the BFD database
-   */
-  static HikariDataSource createDataSource(
```

Comment:
@brianburton why are we removing `createDataSource`?

Reply:
This is a refactoring of the common logic for creating a DataSource object.  Instead of having similar/same logic in multiple places I merged it into the DatabaseUtils class.